sentry-android-replay/src/main/java/io/sentry/android/replay/WindowRecorder.kt

```diff
@@ -55,11 +55,6 @@ internal class WindowRecorder(
             return
         }
 
-//    val (height, width) = (wm.currentWindowMetrics.bounds.bottom /
```

Comment:
I guess this can be safely removed

Reply:
yeah red lines means these have been removed, haha ;)